zerver/openapi/zulip.yaml

```diff
@@ -9507,6 +9507,18 @@ paths:
       operationId: get_server_settings
       summary: Get server settings
       tags: ["server_and_organizations"]
+      x-response-description: |
+        [ldap-auth]: https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#ldap-including-active-directory
```

Comment:
Does this link go somewhere else in the page?  That seems likely.

Reply:
@timabbott That's indeed the case. Moving it to correct location.